bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java

```diff
@@ -443,10 +420,11 @@ private int[] computeRoundTabOutline(int itemIndex, boolean onBottom, int bottom
 		int index = 0;
 		int radius = cornerSize / 2;
 		int circX = bounds.x + radius;
-		int circY = onBottom ? bounds.y + bounds.height + 1 - header - radius : bounds.y - 1 + radius;
+		int selectionX1, selectionY1, selectionX2, selectionY2;
+		int circY = onBottom ? bounds.y + bounds.height - 1 - header - radius : bounds.y + radius;
 		if (itemIndex == 0 && bounds.x == -computeTrim(CTabFolderRenderer.PART_HEADER, SWT.NONE, 0, 0, 0, 0).x) {
 			circX -= 1;
-			points[index++] = circX - radius;
+			points[index++] = circX - radius + 1;
 			points[index++] = bottomY;
 
 			points[index++] = circX - radius;
```

Comment:
Should it be like this?
```suggestion
			points[index++] = selectionX1 = circX + 1 - radius;
```